lib/src/components/NxTabs/NxTab.tsx

```diff
@@ -78,7 +86,7 @@ const NxTab = function NxTabElement(props: NxTabProps) {
     const nextElement = getNextElementFromEventKey(event.key);
 
     if (nextElement) {
-      event.preventDefault();
+      preventDefaultWhenModifierKeyIsNotPressed(event.nativeEvent);
       (nextElement as HTMLElement).focus();
```

Comment:
This will still move the focus on e.g. Shift+ArrowRight.  I'd think we wouldn't want that.  I know for instance it drives me crazy that Sketch reacts to Ctrl+Alt+Right (by changing to an entirely different mockup) because I use that key combination to switch workspaces.  IMO our navigations should only be based on _modifierless_ key events.

Reply:
> This will still move the focus on e.g. Shift+ArrowRight. I'd think we wouldn't want that. I know for instance it drives me crazy that Sketch reacts to Ctrl+Alt+Right (by changing to an entirely different mockup) because I use that key combination to switch workspaces. IMO our navigations should only be based on _modifierless_ key events.

Changed navigations based on modifierless key events here: 56973dc6a829f285e56a1ce4295a8c7d3221040c